Nos.xcodeproj/project.pbxproj

```diff
@@ -2863,8 +2863,9 @@
 				GCC_PREPROCESSOR_DEFINITIONS = "STAGING=1";
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = Nos/Info.plist;
-				INFOPLIST_KEY_CFBundleDisplayName = "Nos Staging";
+				INFOPLIST_KEY_CFBundleDisplayName = "Nos Dev";
```

Comment:
I think Xcode 16 added these INFOPLIST_KEYs to the project settings. They seems safe I guess.

Reply:
@mplorentz here's where the bundle display name broke, I think. Fixed in #1620.

Reply:
weird. Thanks for fixing this.